libavpipe/src/avpipe_xc.c

```diff
@@ -3402,12 +3402,13 @@ avpipe_xc(
     }
 #endif
 
+    int video_stream_index = decoder_context->video_stream_index;
     if (params->xc_type & xc_video) {
         if (encoder_context->format_context->streams[0]->avg_frame_rate.num != 0 &&
```

Comment:
Is this always stream 0?

```
encoder_context->format_context->streams[0]
```

Reply:
Always, we produce one stream in the output: either audio or video. We are not generating two streams at the same time for  the same output.